app/controllers/assignments_controller.rb

```diff
@@ -7,6 +7,9 @@ class AssignmentsController < ApplicationController
   before_action :set_assignment, except: %i[new create]
   before_action :set_unlinked_users, only: [:show]
 
+  SORT_MODES        = ["Assignment acceptance time", "Student name", "Student username"].freeze
+  DEFAULT_SORT_MODE = "Assignment acceptance time"
```

Comment:
What do you think about changing this to `= SORT_MODES.first`? Then we don't need to have this string written twice.